### PR TITLE
Untested - Add links to trase.finance in top navigation

### PR DESCRIPTION
**Pull request by arie-neural-alpha**
Untested - Add links to trase.finance in top navigation

Please test

### CODE REVIEW

File: frontend/styles/_settings.scss [hunk 1]
```diff
@@ -48,11 +48,11 @@ $list-style-image-path: "data:image/svg+xml;utf8,<svg xmlns='http://www.w3.org/2
 // Breakpoints
 $breakpoint-mobile: 425px;
 $breakpoint-tablet: 768px;
-$breakpoint-laptop: 1024px;
+$breakpoint-laptop: 1200px;
```
[the remark this answers is not in the document]
Comment (arie-neural-alpha) [reply frontend/styles/_settings.scss:51]: I searched the repo. and couldn't find that variable, please can you make the change?

[the remark this answers is not in the document]
Comment (Bluesmile82) [reply frontend/styles/_settings.scss:51]: I will add a new PR on the Trase repository adding the link to Trase finance

[the remark this answers is not in the document]
Comment (arie-neural-alpha) [reply frontend/styles/_settings.scss:51]: thank you, I appreciate it.

File: frontend/scripts/router/nav-links.js [hunk 2]
```diff
@@ -55,6 +55,11 @@ let nav = [
   {
     name: 'About',
     page: 'about'
+  },
+  {
+    name: 'Trase Finance',
```
Comment (harrybiddle) [inline frontend/scripts/router/nav-links.js:60]:
```suggestion
    name: 'Finance',
```

This should be just "Finance". Not sure if that affects the widths elsewhere?

**Pull request closed without merging**